apps/src/templates/studioHomepages/RecentCourses.jsx

```diff
@@ -30,15 +41,26 @@ const RecentCourses = React.createClass({
           showLink={showAllCoursesLink}
           isRtl={isRtl}
         >
+          {!isTeacher && studentTopCourse && (
+            <StudentTopCourse
+              isRtl={isRtl}
+              assignableName={studentTopCourse.assignableName}
+              lessonName={studentTopCourse.lessonName}
+              linkToOverview={studentTopCourse.linkToOverview}
+              linkToLesson={studentTopCourse.linkToLesson}
+            />
+          )}
           {topFourCourses.length > 0 && (
             topFourCourses.map((course, index) =>
+            <div key={index}>
               <CourseCard
-                key={index}
                 name={course.name}
                 description={course.description}
                 link={course.link}
                 isRtl={isRtl}
               />
+              {(index % 2 === 0) && <div style={styles.spacer}>.</div>}
```

Comment:
I am a fan of what @islemaster is doing with flexbox and if we make sure this is working on IE11 then I think switching to flexbox will make this stuff much easier going forward.